lib/plugin.js

```diff
@@ -13,6 +13,10 @@ class HTTP {
     this._ky = ky
   }
 
+  getBaseURL (baseURL) {
```

Comment:
i think we don't need to get `baseURL` parameter 
```suggestion
  getBaseURL () {
```